spp_change_request_add_children_demo/models/change_request_add_children.py

```diff
@@ -260,21 +260,25 @@ def update_live_data(self):
             ]
         else:
             uid_rec = None
-        individual_id = self.env["res.partner"].create(
-            {
-                "is_registrant": True,
-                "is_group": False,
-                "name": self.full_name,
-                "family_name": self.family_name,
-                "given_name": self.given_name,
-                "addl_name": self.addl_name,
-                "birth_place": self.birth_place,
-                "birthdate_not_exact": self.birthdate_not_exact,
-                "birthdate": self.birthdate,
-                "gender": self.gender,
-                "phone_number_ids": phone_rec,
-                "reg_ids": uid_rec,
-            }
+        individual_id = (
+            self.env["res.partner"]
+            .sudo()
+            .create(
+                {
+                    "is_registrant": True,
+                    "is_group": False,
+                    "name": self.full_name,
+                    "family_name": self.family_name,
+                    "given_name": self.given_name,
+                    "addl_name": self.addl_name,
+                    "birth_place": self.birth_place,
+                    "birthdate_not_exact": self.birthdate_not_exact,
+                    "birthdate": self.birthdate,
+                    "gender": self.gender,
+                    "phone_number_ids": phone_rec,
+                    "reg_ids": uid_rec,
+                }
+            )
```

Comment:
@emjay0921 Please revert this change, as much as possible do not use sudo() since it bypass the security the the ir.model.access.csv provides.

Upon checking the error in the ticket, it seems that the cause of issue is that all of the change request modules (aside from `Farmer Registry: Add Farmer Change Request`) is not compatible to the module `Farmer Registry: Base` because of the overriding of the `create` and `update` functions that you did in the file spp_farmer_registry_base/models/farm.py

My suggestion to this is that you either make all of the change request compatible to the `Farmer Registry: Base` module or discuss with @anthonymarkQA to add this in the limitations of change requests modules.